.travis.yml

```diff
@@ -39,16 +42,12 @@ before_script:
     else
       composer require mongofill/mongofill --ignore-platform-reqs;
     fi
-  - sudo apt-get update
-  - sudo apt-get install -y libssh2-1-dev vsftpd
-  - sudo useradd -d /home/./gaufrette gaufrette # Removing "./" from this path would break any further Travis builds
-  - echo "gaufrette:gaufrette" | sudo chpasswd
-  - sudo mkdir /home/gaufrette /srv/empty
-  - sudo chown gaufrette:gaufrette /home/gaufrette /srv/ftp
-  - sudo cp .travis/vsftpd.conf /etc/vsftpd.conf
-  - sudo service vsftpd restart
-  - composer self-update && composer install --prefer-source --no-interaction
-  - if [[ ${TRAVIS_PHP_VERSION:0:1} == "5" ]]; then yes | pecl install ssh2 && composer require herzult/php-ssh; fi
+  - docker run -p ${SFTP_PORT}:22 -d atmoz/sftp:alpine ${SFTP_USER}:${SFTP_PASSWORD}:::${SFTP_BASE_DIR}
+  - docker run -d --name ftpd_server -p ${FTP_PORT}:21 -p 30000-30009:30000-30009 -e "PUBLICHOST=${FTP_HOST}" stilliard/pure-ftpd
```

Comment:
Could you update the `docker-compose` and/or Dockerfiles in order to have exactly the same config locally as Travis please?

Reply:
I'll do it next week.